Simplenote/AuthViewController+Swift.swift

```diff
@@ -62,21 +61,18 @@ extension AuthViewController {
     }
 
     func refreshButtonTitles() {
-        let actionText    = signingIn ? Localization.signInAction   : Localization.signUpAction
-        let tipText       = signingIn ? Localization.signUpTip      : Localization.signInTip
-        let switchText    = signingIn ? Localization.signUpAction   : Localization.signInAction
-
-        actionButton.title         = actionText
-        switchTipField.stringValue = tipText.uppercased()
-        switchActionButton.title   = switchText.uppercased()
+        actionButton.title          = mode.primaryActionText
+        secondaryActionButton.title = mode.secondaryActionText?.uppercased() ?? ""
+        switchTipField.stringValue  = mode.switchActionTip.uppercased()
+        switchActionButton.title    = mode.switchActionText.uppercased()
```

Comment:
Rather than depending on the `signingIn` boolean, the UI's strings are determined by the `AuthenticationMode` object